bin/prettier.js

```diff
@@ -145,7 +146,10 @@ function getTrailingComma() {
   }
 }
 
+const r = require;
 const options = {
+  astTransform: argv["ast-transform"] &&
```

Comment:
I wanted to leave room for a future `docTransform` à la https://github.com/prettier/prettier/issues/1550. This would allow e.g. [`prettier-elastic-vars`](https://github.com/medikoo/prettier-elastic-vars) to be implemented without having to use `prettier.__debug`, or by simply passing a module path on the CLI (`prettier --doc-transform path/to/doc/transform`).